packages/@dcl/sdk/package.json

```diff
@@ -9,7 +9,6 @@
     "@dcl/explorer": "1.0.164182-20240702173148.commit-d4a0e99",
     "@dcl/js-runtime": "file:../js-runtime",
     "@dcl/react-ecs": "file:../react-ecs",
-    "@dcl/sdk-commands": "file:../sdk-commands",
```

Comment:
We don't actually need the `@dcl/sdk-commands` for `@dcl/sdk` itself, we only need it to execute the build steps, as a wrapper around our projects.
Keeping the `sdk-commands` available in a global install works just as well (or even via `npx` if we're always wanting the latest version), and avoids having to keep a copy of the `@dcl/sdk-commands` in every single one of the `node_modules` folders of our projects.
If a project is locked to a previous version of the `@dcl/sdk`, then the best option is to require listing the `@dcl/sdk-commands` as well in the project's `package.json`.
In the end, there is a circular dependency to remove, and doing it this way seems like the cleanest.

Reply:
Ok that's not true.
We need sdk-commands inside the @dcl/sdk package since that package is what the scenes only knows.
Without this, every scene would need to install separately the @dcl/sdk-commands package to run, start, deploy each project.

Closing this issue since is a NO go for our current implementation.

Reply:
I'm sorry but in for standard development environment, what I wrote is _very much_ true. The main SDK should **not** have dependencies on the tool which will be calling the various modules which will be depending on the SDK itself !

Glad to see that the **circular dependency** at least will finally be fixed in https://github.com/decentraland/asset-packs/pull/118 😊